platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/SecurityUtils.java

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration.migration;
+
+import java.security.AccessController;
+import java.security.PrivilegedActionException;
+import java.security.PrivilegedExceptionAction;
+import org.apache.commons.lang.Validate;
+import org.codice.ddf.util.function.ThrowingRunnable;
+import org.codice.ddf.util.function.ThrowingSupplier;
+
+/** Utility classes to help with the security manager. */
+public class SecurityUtils {
```

Comment:
❓ Would it make sense to rename and move this into something lower level, such as `platform-util`?

✏️ Regardless of whether moving it makes sense, I would strongly recommend giving it a different name such that ambiguity is removed. The first time I saw this class accessed earlier in this review, I thought it was the extant `SecurityUtils`.

Reply:
It cannot be moved out of this library otherwise the privileges that are extended would be the one from a different library and not from this one.

Renamed to AccessUtils